tests/acceptance/features/coreApiSharePublicLink1/changingPublicLinkShare.feature

```diff
@@ -131,3 +131,69 @@ Feature: changing a public link share
     When the public deletes file "parent.txt" from the last public link share using the password "%public%" and new public WebDAV API
     Then the HTTP status code should be "403"
     And as "Alice" file "PARENT/parent.txt" should exist
+
+
+  Scenario Outline:try to remove password of a public link share(change, create permission) as a normal user
```

Comment:
we used to write `<actor> does something`. maybe this way the titles can be clearer.
```suggestion
  Scenario Outline: normal user tries to remove password of a public link share (change, create permission)
```